modules/processing/static.py

```diff
@@ -1244,11 +1314,19 @@ def _parse(self, filepath):
         if "author" in metadata:
             pdfresult["Info"]["Author"] = convert_to_printable(self._clean_string(metadata["author"]))
 
+        if pdfresult["Keywords"].get("/JS", 0) > 0 or pdfresult["Keywords"].get("/JavaScript", 0) > 0:
+            pdfresult["Info"]["JavaScript"] = "True"
+
         if len(urlset):
             pdfresult["JS_URLs"] = list(urlset)
         if len(annoturiset):
             pdfresult["Annot_URLs"] = list(annoturiset)
 
+        if HAVE_PDFMINER:
+            mined_urls = set((self.base_uri + url.decode() for url in self._mine_for_urls(self.file_path)))
+            if len(mined_urls):
```

Comment:
why do we need the len?
```suggestion
            if mined_urls:
```